package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@quintype/framework",
-  "version": "4.0.7",
+  "version": "4.1.0",
```

Comment:
How did you upgrade from a branch? Feature upgrades should be done only on master.